.github/workflows/update.yml

```diff
@@ -27,8 +29,19 @@ jobs:
         env:
           GITHUB_TOKEN: "${{ secrets.OCTOKITBOT_PAT }}"
         with:
-          title: "\U0001F6A7 \U0001F916\U0001F4EF Webhooks changed"
-          body: "A new release of [@octokit/webhooks-definitions](https://github.com/octokit/webhooks) was just released \U0001F44B\U0001F916\n\nThis pull request updates the TypeScript definitions derived from `@octokit/webhooks-definitions`. I can't tell if the changes are fixes, features or breaking, you'll have to figure that out on yourself and adapt the commit messages accordingly to trigger the right release, see [our commit message conventions](https://github.com/octokit/openapi/blob/main/CONTRIBUTING.md#merging-the-pull-request--releasing-a-new-version).\n"
+          title: "🚧 🤖📯 Webhooks changed"
+          body: "A new release of
```

Comment:
Are you sure this won't cause any problems with the PR body

Reply:
Yep, this is an automated PR and probably when parsing and dumping back the yaml file the encoding was messed up :'(